src/ui/FacetValueSuggestions/FacetValueSuggestions.ts

```diff
@@ -143,9 +143,13 @@ export class FacetValueSuggestions extends Component {
   private queryStateFieldFacetId;
 
   static defaultTemplate(this: FacetValueSuggestions, row: IFacetValueSuggestionRow, omnibox: Omnibox): string {
-    const keyword = DomUtils.highlightElement(row.keyword, omnibox.getText(), 'coveo-omnibox-hightlight2');
-    const facetValue = DomUtils.highlightElement(row.value, row.value, 'coveo-omnibox-hightlight');
-    const details = this.options.displayEstimateNumberOfResults ? ` (${l('ResultCount', row.numberOfResults.toString())})` : '';
+    const keyword = DomUtils.highlightElement(row.keyword, omnibox.getText(), 'coveo-omnibox-suggestion-input-corrected');
+    const facetValue = DomUtils.highlightElement(row.value, row.value, 'coveo-omnibox-suggestion-input');
+    const details = this.options.displayEstimateNumberOfResults
+      ? `<span class="coveo-omnibox-suggestion-results-count">
+        (${l('ResultCount', row.numberOfResults.toString(), row.numberOfResults)})
```

Comment:
Pretty cool using a template literal here. 
I would only at least extract the string `l('ResultCount', row.numberOfResults.toString(), row.numberOfResults)` to a separate variable. You could even export the whole `details` to a method

Reply:
Good idea, I simplified it by adding a new method to `DomUtils` called `highlight`.